src/app/services/project.service.ts

```diff
@@ -1,45 +1,43 @@
 import { Injectable } from '@angular/core';
+import { Headers, RequestOptions, Response, URLSearchParams } from '@angular/http';
+import { Observable } from 'rxjs';
 
-import { Project } from '../models/project';
+import { ProjectData, APIProject } from '../models/project';
+import { ApiHttp } from '../auth/api-http.service';
+import { apiHost } from "../constants";
+
+import * as _ from 'lodash';
 
 @Injectable()
 export class ProjectService {
 
-    private storageKey: string = 'cclab.projects';
-    private projects: Project[] = [];
-    private storage: Storage = window.localStorage;
-
-    constructor() {
-        this.projects = JSON.parse(this.storage.getItem(this.storageKey)) || [];
-    }
+    private projects: APIProject[] = [];
 
-    add(project: Project) {
-        this.projects.push(project);
-        this.save();
-    }
+    constructor(private apiHttp: ApiHttp) {}
 
-    get(id: string) {
-        return this.projects.find(project => project.id === id);
+    add(project: APIProject): Observable<APIProject> {
```

Comment:
Bah. These methods should have always returned Observables to make this local storage -> API change easier.